src/lib-components/lists/DynamicTable.vue

```diff
@@ -306,6 +308,7 @@ loadAndRender()
           <tr
             v-for="(row, rowIdx) in rows"
             :key="rowIdx"
+            class="even:bg-gray-50"
```

Comment:
I'm good with making this our default, though I subjectively think we could drop the border between rows when they are striped.  The extra border adds to the visual competition.  Archive uses tables quite a bit more than Portal, so @profsmallpine should probably weight in on moving all in on striped tables.

I'd expect we'd move the `DataTable` to the same style as well for consistency.

Reply:
I don't find one as more compelling than the other, but I think it is best to pick one, either using border or stripes. The downside of stripes for me is that it makes even rows the same background color as the header which I find visually noisy. Typically, the headers will loose their background shading so that you put the visual emphasis on the differentiation of rows.

Reply:
How do we feel about this option?

Striped, no border between rows, header to `bg-gray-100` from `bg-gray-50`.
<img width="1204" alt="Screenshot 2024-04-23 at 11 02 49 AM" src="https://github.com/xy-planning-network/trees/assets/27823069/e17f6ff7-ed24-4186-8a86-1da40802fb3e">

Reply:
Yep, should be OK.  We use `bg-gray-100` as the app shell background color, so outside a card wrapper it _might_ be a touch off, but not to a degree I think we need to sweat right now.  I think Portal currently wraps all tables in some kind of card like wrapper.